winter-usecases/src/main/java/de/uni_mannheim/informatik/dws/winter/usecase/movies/Movies_DuplicateBasedSchemaMatching.java

```diff
@@ -84,10 +84,16 @@ public double compare(Attribute a1, Attribute a2,
 					}
 
 					@Override
-					public Map<ComparatorDetails, String> getComparisonResult() {
+					public ComparatorLogger getComparisonLog() {
```

Comment:
Create default implementation in MatchingRule, s.t. matching rules which do not implement logging don't need to specify this method (same for setComparisonLog)

Reply:
Done